spec/factories.rb

```diff
@@ -98,28 +98,118 @@
     candidate
```

Comment:
Just a general question and not really related to this PR, what was the thinking behind having a single factories file  vs having them in separate files as suggested in [best practices](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#best-practices) in `FactoryBot`?

Reply:
➕

Reply:
An early convention of the project, intended (IIRC) to encourage keeping factory definitions simple. Worth reviewing, maybe at the next tech debt review.